src/types/depositERC20Parameters.ts

```diff
@@ -0,0 +1,9 @@
+import { Address, Hex } from 'viem'
+
+export type DepositERC20Parameters = {
```

Comment:
I am open to these each living in their own file, but I wonder it's best 🤔

Reply:
I think if it gets out of hand we could merge them down the road